package.json

```diff
@@ -134,8 +135,8 @@
     "watch": "WATCH=1 ./scripts/test/js_test.sh",
     "repl": "node --require ./scripts/repl.js",
     "flow": "flow check",
-    "fmt": "LOG_LEVEL= prettier-eslint --write --no-semi --ignore 'static/js/flow/**/*.js' 'static/js/**/*.js'",
-    "fmt:check": "LOG_LEVEL= prettier-eslint --list-different --no-semi --ignore 'static/js/flow/**/*.js' 'static/js/**/*.js'"
+    "fmt": "LOG_LEVEL= prettier-eslint --write --no-semi --ignore $PWD/'static/js/flow/**/*.js' $PWD/'static/js/**/*.js'",
```

Comment:
What's the reason for adding `$PWD` here? Are we changing the working directory?

Reply:
i have updated `prettier-eslint` that's what it requires for the latest version to provide $PWD that's what i added because of getting PATH issue in the build.

Reply:
see that screenshot because of which I had added for this.
<img width="1412" alt="Screenshot 2020-04-14 at 19 52 54" src="https://user-images.githubusercontent.com/4043989/79239365-ecff6000-7e89-11ea-86e6-9c5f2eccc065.png">